tests/protocol/stun/stun.zeek

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 by the Zeek Project. See LICENSE for details.
+
+# @TEST-EXEC: set >envs
+# @TEST-EXEC: ${ZEEK} -NN >zeek 2>&1
```

Comment:
This line currently does nothing and looks like a debugging leftover. Let's add a proper `availability` check like we do for other analyzers.

Reply:
I copied this from @0xxon Wireguard analyzer.

Reply:
That is true :).

At least when I wrote wireguard - I also just stole it from another test because it made debugging easier.